Formula/asdf.rb

```diff
@@ -29,10 +29,16 @@ def install
 
   def caveats; <<~EOS
     Add the following line to your bash profile (e.g. ~/.bashrc, ~/.profile, or ~/.bash_profile)
-         source #{opt_prefix}/asdf.sh
+    If you are using a framework, such as oh-my-zsh, place them at the end of your .zshrc
```

Comment:
Shouldn't these be handled by the regular completion instructions ? See https://docs.brew.sh/Shell-Completion